src/components/Grid/_grid.scss

```diff
@@ -0,0 +1,56 @@
+// ----------------------------------
+// UI Components
+// Grid
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+/* Settings */
+
+@import 'grid-settings';
+
+/* Mixins */
+
+@import 'grid-mixins';
+
+/* Styles */
+
+.grid {
+  display: grid;
+  // stylelint-disable value-no-vendor-prefix
+  display: -ms-grid;
+  @include grid-container;
+
+  // Columns: 6 / Gutter: 12px
+  @include grid-columns($grid-gutter-xs, grid-repeat(6, 1fr));
+  @include grid-rows(0, 1fr);
+
+  @include mq(sm) {
+    // Columns: 12 / Gutter: 20px
+    @include grid-columns($grid-gutter-sm);
+  }
+
+  @include mq(md) {
+    // Columns: 12 / Gutter: 40px
+    @include grid-columns($grid-gutter-md);
+  }
+
+  @include mq(xl) {
+    // Columns: 12 / Gutter: 48px
+    @include grid-columns($grid-gutter-xl);
+  }
+}
+
+// Reset nested grids
+.grid--nested {
+  margin-left: 0;
+  margin-right: 0;
+  @include grid-column(1, 13);
+
+  > * {
+    @include grid-column(1, 13);
+  }
+}
+
+@import 'Cell/grid-cell';
```

Comment:
Not sure if this is a good idea - the capital letter bugs me. Alternatives:

- moving the GridCell component to `components/` root
- moving the `_grid-cell` styles to Grid root
- moving the `_grid-cell` styles to the `_grid` file
- @importing directly in to `30-components/__manifest`